packages/komencikit/examples/attestation-flow.ts

```diff
@@ -0,0 +1,157 @@
+import { base64ToHex } from '@celo/base'
+import {
+  printAndIgnoreRequestErrors,
+  requestAttestationsFromIssuers,
+} from '@celo/celotool/lib/lib/attestation'
+import { ContractKit } from '@celo/contractkit'
+import { LocalWallet } from '@celo/contractkit/lib/wallets/local-wallet'
+import Web3 from 'web3'
+import { KomenciKit } from '../src'
+
+const WALLET_IMPLEMENTATION_ADDRESS = '0x88a2b9B8387A1823D821E406b4e951337fa1D46D'
+
+const wallet = new LocalWallet()
+const pkey = Web3.utils.randomHex(32)
+// const pkey = '0xdc771e7878396744e17afcb0dea4cfc96ce6f116107c7bcc0687b812048a2bf7'
+wallet.addAccount(pkey)
+console.log('Private key: ', pkey)
+
+const provider = new Web3.providers.HttpProvider('https://alfajores-forno.celo-testnet.org')
+const web3 = new Web3(provider)
+const contractKit = new ContractKit(web3, wallet)
+const account = wallet.getAccounts()[0]
+console.log('Account: ', account)
+const komenciKit = new KomenciKit(contractKit, account, {
+  url: 'http://localhost:3000',
+})
+
+const readline = require('readline')
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+})
+
+const run = async () => {
+  const attestations = await contractKit.contracts.getAttestations()
+  console.log('Attestations: ', attestations.address)
+  console.log('Starting')
+  const startSession = await komenciKit.startSession('captcha-token')
+
+  console.log('StartSession: ', startSession)
+  if (!startSession.ok) {
+    return
+  }
+  const deployWallet = await komenciKit.deployWallet(WALLET_IMPLEMENTATION_ADDRESS)
+  console.log('DeployWallet: ', deployWallet)
+  if (!deployWallet.ok) {
+    return
+  }
+  const walletAddress = deployWallet.result
+  // cached:
+  const phoneNumber = '+40723301264'
+  // const identifier = '0x59c637d9c774d242dc36bdb0445e29fa79c69b74ef70b6d1a5c407c1db6b4110'
+  const getIdentifier = await komenciKit.getDistributedBlindedPepper(phoneNumber, 'unknown')
+  console.log('GetIdentifier: ', getIdentifier)
+  if (!getIdentifier.ok) {
+    return
+  }
+  const identifier = getIdentifier.result.identifier
+  const pepper = getIdentifier.result.pepper
+  const statsBefore = await attestations.getAttestationStat(identifier, walletAddress)
+  console.log('Before ===== ')
+  console.log(statsBefore)
+  const requestAttestations = await komenciKit.requestAttestations(identifier, walletAddress, 3)
+  const statsAfter = await attestations.getAttestationStat(identifier, walletAddress)
+  console.log('After ===== ')
+  console.log(statsAfter)
+  console.log('RequestAttestations: ', requestAttestations)
+  if (!requestAttestations.ok) {
+    return
+  }
+
+  const events = await attestations.getPastEvents('AttestationsRequested', {
```

Comment:
Can we make these a constant enum defined somewhere? 
`AttestationsRequested` `AttestationIssuerSelected` `WalletDeployed`

Reply:
Yeah it's one thing I wanted to look into. Theoretically they should be autogenerated somewhere. I was expecting to see them on the contract wrapper but either that's not expected or there's something weird going on in the solidity->generated types->contract->wrapper. But I'll look into this some more.